Frameworks/UIKit/StarboardXaml/StarboardXaml.cpp

```diff
@@ -165,6 +165,11 @@ void App::OnActivated(IActivatedEventArgs^ args) {
     UIApplicationActivated(args);
 }
 
+void App::OnFileActivated(FileActivatedEventArgs^ args)
```

Comment:
You need to also edit  msvc\vsimporter-templates\WinStore10-App\App.xaml.cpp and msvc\vsimporter-templates\WinStore10-App\App.xaml.h file